### PR TITLE
Add TrafficOps login dispersion

### PR DESCRIPTION
**Pull request by jrushford**
Optimizing TrafficOps logins.

- Adds TrafficOps login dispersion
- Adds session re-use to limit TrafficOps logins.

## Which Traffic Control components are affected by this PR?

- Traffic Control Health Client (tc-health-client)

## PR submission checklist
- [x] This PR has tests 
- [x] This PR has documentation 
- [x] This PR has a CHANGELOG.md entry 
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

**Review by jpappa200 — APPROVED** [review]
LGTM 👍

File: tc-health-client/README.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ Sample configuarion file:
     "to-request-timeout-seconds": "5s",
     "tm-poll-interval-seconds": "60s",
     "tm-proxy-url", "http://sample-http-proxy.cdn.net:80",
-    "tm-update-cycles": 5,
+    "to-to-login-dispersion-factor": 90,
```
Comment (rawlinp) [inline tc-health-client/README.md:107]: `to-to-login`, one `to` too many

Comment (jrushford) [reply tc-health-client/README.md]: @rawlinp fixed it

**Comment by jrushford** [comment]
> just noticed this, since this is a long-running service, you can save the session and reuse it. That way it will reuse the cookie with every request, and if it gets 401'd due to the cookie expiring, the client will automatically try to login again to refresh it.
> 
> As long as a request is made at least once an hour, the cookie should stay fresh too.

Thanks @rawlinp I was talking to @jpappa200 and he had added the cookie reuse in t3c, he was going to add it to the health-client as well.

**Review by jpappa200 — APPROVED** [review]
Looks good 👍

**Review by rawlinp — APPROVED** [review]

**Pull request merged**